README.md

```diff
@@ -155,3 +161,19 @@ autocmd({ 'ModeChanged' }, {
 
 https://user-images.githubusercontent.com/54583542/220056425-a7698013-7173-4247-9d40-d468b24df47a.mov
 
+
+### How do I add icons (signs) for mode's last positions?
+
+You can decide which modes should be added. 
+Define the `smoothcursor_<mode>` sign group and set your icon.
+Each `<mode>` is a lowercase letter, e.g., `n` for normal mode, `i` for insert mode, etc.
+
+**example**
+```lua
+vim.fn.sign_define('smoothcursor_v', { text = ' ' })
+vim.fn.sign_define('smoothcursor_V', { text = '' })
+vim.fn.sign_define('smoothcursor_i', { text = '' })
+vim.fn.sign_define('smoothcursor_�', { text = '' })
```

Comment:
Would it be more reasonable to use `smoothcursor_^v` instead?

Reply:
Probably yes. I don't really use this block.. visual.. selection thing - I didn't even know vim had something like this until I searched for the icons :O so yeah, "^v" is probably better. Did you test this? I don't know if "^v" is the actual mode name

Reply:
You can check out all the return values with `:h mode()`. According to this document, Visual blockwise is expressed as `CTRL-V`, so maybe `smoothcursor_^V` or `smoothcursor_CTRL_V` would be more appropriate. But of course, you'd need to convert `` to `CTRL_V` in some places.